sdk/identity/identity/test/manual-integration/AzureFunctions/IdentityTest/KeyvaultAuthentication/index.ts

```diff
@@ -0,0 +1,32 @@
+import { SecretClient } from "@azure/keyvault-secrets";
+import { DefaultAzureCredential } from "@azure/identity";
+import { AzureFunction, Context, HttpRequest } from "@azure/functions";
+
+const httpTrigger: AzureFunction = async function(
+  context: Context,
+  req: HttpRequest
+): Promise<void> {
+  try {
+    const credential = new DefaultAzureCredential();
+
+    const vaultUri = process.env.KEYVAULT_URI;
+
+    if (!vaultUri) {
+      throw new Error("Missing KEYVAULT_URI environment variable.");
+    }
+    const client = new SecretClient(vaultUri, credential);
+
+    await client.setSecret("secret-name-system", "secret-value-system");
```

Comment:
Are you also trying to demo reading from the http context as well so you can use the request to populate the secret?

Reply:
Not Karishma of course 😄 but this is actually not meant for consumption by users but by our testing team to manually test azure identity in azure functions - Not sure if that changes your review at all but figured it's worth mentioning. The goal here is to have something that the testing team can run manually and eventually automate (I think)

Anyway, that's how I understood it at least

Reply:
Oh dang it, sorry about that @KarishmaGhiya and @maorleger  - I did not realize it was for internal use only.

I think most of it just best practice stuff or mild things, so please feel free to ignore! :)